include/GafferCycles/IECoreCyclesPreview/IECoreCycles.h

```diff
@@ -47,6 +47,10 @@ IECORE_POP_DEFAULT_VISIBILITY
 namespace IECoreCycles
 {
 
+/// Returns the value of the `CYCLES_ROOT` environment variable.
+/// Throws if the variable is not set.
```

Comment:
The implementation is actually returning "" rather than throwing.

Reply:
Right! I had debated which to do and that comment got stale. Fixed in https://github.com/GafferHQ/gaffer/pull/5761/commits/b5834f697098d2cc0476d237b3553a281716f0b0